src/web/mjs/connectors/OnePunchMan.mjs

```diff
@@ -0,0 +1,46 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class OnePunchMan extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'onepunchman';
+        super.label = 'One-Punch Man';
+        this.tags = [ 'manga', 'english', 'high-quality' ];
+        this.url = 'https://ww3.one-punchman.com/';
+    }
+
+    async _getMangas() {
+        return [
+            {
+                id: 'onepunchman',
+                title: this.label
+            }
+        ];
+    }
+
+    async _getChapters() {
+        let request = new Request(this.url, this.requestOptions);
```

Comment:
Same here, include the manga identifier, e.g.
`new Request(new URL(manga.id, this.url), this.requestOptions)`